content/blog/2016/2016-05-04-the-state-of-JAM

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "The State of Jenkins Area Meetup (JAM)"
+tags:
+- meetup
+- JAM
+- event
+author: alyssat
+---
+
+NOTE: _This is a guest post by link:https://github.com/alyssat[Alyssa Tong].
```

Comment:
@oleg-nenashev that's not what I'm saying. My understanding is that @alyssat wrote the blog post and asked @hinman55 to go through the publishing process. That's silly, but totally fine.